### PR TITLE
fix: remove row focus background color

### PR DESCRIPTION
**Pull request by ling1726**
Removes the the row focus background so that focus state in the teams high contrast theme appears clearly

### Light
image

### Dark
image

### Teams HC 
image

### Light

image

### Dark
image

### Teams HC 
image

Fixes https://github.com/microsoft/fluentui/issues/29033

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/Avatar-mount_base.html">617</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/Avatar-mount.html">637</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/Button-mount_base.html">308</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/Button-mount.html">323</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/Field-mount_base.html">1114</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/Field-mount.html">1151</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/FluentProvider-mount_base.html">704</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/FluentProvider-mount.html">697</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/FluentProviderWithTheme-mount_base.html">78</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/FluentProviderWithTheme-mount.html">74</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">75</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">66</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">69</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">64</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/InfoButton-mount_base.html">7</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/InfoButton-mount.html">9</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/MakeStyles-mount_base.html">888</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/MakeStyles-mount.html">897</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/Persona-mount_base.html">1795</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/Persona-mount.html">1695</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/SpinButton-mount_base.html">1356</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29364/perf-test-react-components/SpinButton-mount.html">1432</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                                     |     Baseline (minified/GZIP) |                            PR |                                                                                                                                                                                                    Change |
| :-------------------------------------------------------------------------------------------------------------------- | ---------------------------: | ----------------------------: | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/DataGrid.fixture.js'>DataGrid</abbr>                         | `155.92 kB`<br />`43.463 kB` | `155.809 kB`<br />`43.458 kB` |  `-111 B` <br />`-5 B`  |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/PrimitivesOnly.fixture.js'>Table (Primitives only)</abbr>    | `42.715 kB`<br />`13.341 kB` |  `42.604 kB`<br />`13.348 kB` |   `-111 B` <br />`7 B`  |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableAsDataGrid.fixture.js'>Table as DataGrid</abbr>         | `129.19 kB`<br />`34.771 kB` | `129.079 kB`<br />`34.763 kB` |  `-111 B` <br />`-8 B`  |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSelectionOnly.fixture.js'>Table (Selection only)</abbr> | `74.557 kB`<br />`20.071 kB` |  `74.446 kB`<br />`20.054 kB` | `-111 B` <br />`-17 B`  |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSortOnly.fixture.js'>Table (Sort only)</abbr>           |  `73.164 kB`<br />`19.67 kB` |  `73.053 kB`<br />`19.654 kB` | `-111 B` <br />`-16 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `69.227 kB`<br />`19.584 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `206.783 kB`<br />`59.085 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `40.793 kB`<br />`13.521 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |     `6.503 kB`<br />`2.22 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/20ce41b1804af616985f527761602d49a9d58dd2'>20ce41b1804af616985f527761602d49a9d58dd2</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

File: packages/react-components/react-table/src/components/TableRow/useTableRowStyles.styles.ts [hunk 1]
```diff
@@ -57,7 +58,7 @@ const useStyles = makeStyles({
   noAppearanceFocusWithin: {
     ...createCustomFocusIndicatorStyle(
       {
-        backgroundColor: tokens.colorSubtleBackgroundHover,
+        // backgroundColor: tokens.colorSubtleBackgroundHover,
```
Comment (layershifter) [inline packages/react-components/react-table/src/components/TableRow/useTableRowStyles.styles.ts:61]: Cruft? Do we need to call `createCustomFocusIndicatorStyle` at all?

Comment (ling1726) [reply packages/react-components/react-table/src/components/TableRow/useTableRowStyles.styles.ts:61]: Sorry, the PR was missing a commit - my bad 🙈

**Review by layershifter — APPROVED** [review]

**Review by miroslavstastny — APPROVED** [review]

**Pull request merged**